### PR TITLE
MOMZA-111 Add screens to registration flow

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
_Consent screen_
An additional USSD screen needs to be added to the different registration use cases. This screen will request consent from mothers for their data to be collected and used by the MomConnect service, along with consenting to receive messages on public holidays and weekends.

_On selection of an option:_
The user’s response will be stored against their MSISDN as an extra contact field titled Consent.
Consent will be stored as text and not the corresponding menu item (e.g. ‘Consent’ not ‘1’.)

The different registration use cases (found here) should be updated as follows:

_Clinic Registration_
The Consent screen should be added before: 4 - Enter Clinic code. 
When providing consent, users will be directed to 4 - Enter Clinic code
If a user doesn’t provide consent, they will be directed to “You have chosen not to receive MomConnect SMSs”

_CHW suspected pregnancy registration_
Add a USSD screen before: 3 - type of identification.
When providing consent, users will be directed to 3 - type of identification.
If a user doesn’t provide consent, they will be directed to “You have chosen not to receive MomConnect SMSs”
_Public Subscription_
Add the Consent screen before: 2 - suspected pregnancy. 
When providing consent, users will be directed to 4 - type of identification. 
If a user doesn’t provide consent, they will be directed to “You have chosen not to receive MomConnect SMSs”
The following should be removed from screen 4 - type of identification:
We need to collect some info to send you messages. This is private and will only be used to help you if you go to a clinic.

Also change: "We are sorry but this service is only for pregnant mothers. If you have other health concerns please visit your nearest clinic." to "You have chosen not to receive MomConnect SMSs” on the Public line.

Copy for these screens is here: https://docs.google.com/spreadsheets/d/1syJ55kirTWkO5eldmdC8yGEOu_-jat03tjgB-yAz9Po/edit#gid=0

Complete POPI scope is here: https://docs.google.com/document/d/1TLiJytIDA3UCEAE-gPl0XSD9ZaPnpY51-5DhOR5lx0M/edit#heading=h.iv304ga2c78q

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: src/utils.js [hunk 1]
```diff
@@ -740,6 +740,7 @@ go.utils = {
             mom_edd: go.utils.get_edd(im, contact),  // 'YYYY-MM-DD' | null
             mom_id_no: go.utils.get_identification_no(contact),
             mom_dob: contact.extra.dob || null,  // 'YYYY-MM-DD' | null
+            consent: contact.extra.consent || null, // 'true' | null
```
Comment (imsickofmaps) [inline src/utils.js]: I don't think the spec called for the data to be saved to the registration. Registration can not occur without consent.

Comment (imsickofmaps) [inline src/utils.js]: Looks like it does. This is now fine.

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas please review last commit

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps 👍

**Pull request merged**